.github/workflows/tests.yml

```diff
@@ -56,8 +56,9 @@ jobs:
       - uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - run: pip install -r requirements-tests-py3.txt
-      - run: ./tests/pytype_test.py
+      - run: |
+          pip install -r requirements-tests-py3.txt
+          ./tests/pytype_test.py
```

Comment:
I would prefer these as separate steps. That way it's easier to see which command failed. This also applies to `mypy:` job.